### PR TITLE
test(mdc-tabs): add performance tests for mdc-tabs

### PR DESCRIPTION
**Pull request by wagnermaciel**

### CODE REVIEW

File: test/benchmarks/mdc/tabs/tabs.perf-spec.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      params: [],
+      setup: async() => await $('#show-three-tabs').click(),
+      prepare: async() => await $('#mat-tab-label-0-0').click(),
+      work: async() => await $('#mat-tab-label-0-1').click(),
+    });
+  });
+
+  it('paginates tabs', async() => {
+    async function isTabPaginatorDisabled(ele: ElementFinder) {
+      return (await ele.getAttribute('class')).includes('mat-tab-header-pagination-disabled');
```
Comment (devversion) [inline test/benchmarks/mdc/tabs/tabs.perf-spec.ts:64]: It looks like the class in the MDC tabs is different: `mat-mdc-tab-header-pagination-disabled`.

File: test/benchmarks/mdc/tabs/tabs.perf-spec.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      id: 'tab-pagination',
+      url: '',
+      ignoreBrowserSynchronization: true,
+      params: [],
+      prepare: async() => {
+        await $('#hide').click();
+        await $('#show-twenty-tabs').click();
+      },
+      work: async() => {
+        const nextBtn = $('.mat-tab-header-pagination-after');
```
Comment (devversion) [inline test/benchmarks/mdc/tabs/tabs.perf-spec.ts:76]: Similarly the class for the pagination button seems different: `mat-mdc-tab-header-pagination-after`.

File: test/benchmarks/mdc/tabs/app.module.ts [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  selector: 'app-root',
+  template: `
+  <button id="show-three-tabs" (click)="showThreeTabs()">Show Three Tabs</button>
+  <button id="show-ten-tabs" (click)="showTenTabs()">Show Ten Tabs</button>
+  <button id="show-twenty-tabs" (click)="showTwentyTabs()">Show Twenty Tabs</button>
+  <button id="hide" (click)="hide()">Hide</button>
+
+  <mat-tab-group *ngIf="areThreeTabsVisible">${threeTabs}</mat-tab-group>
+  <mat-tab-group *ngIf="areTenTabsVisible">${tenTabs}</mat-tab-group>
+  <mat-tab-group *ngIf="areTwentyTabsVisible">${twentyTabs}</mat-tab-group>
```
Comment (devversion) [inline test/benchmarks/mdc/tabs/app.module.ts:27]: Optional Nit: Indent by two spaces.

**Review by devversion — COMMENTED** [review]
cc. @crisbeto as he built MDC tabs IIRC.

File: test/benchmarks/mdc/tabs/app.module.ts [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+@NgModule({
+  declarations: [TabsBenchmarkApp],
+  imports: [
+    BrowserModule,
+    BrowserAnimationsModule,
+    MatTabsModule,
+  ],
+  providers: [],
```
Comment (crisbeto) [inline test/benchmarks/mdc/tabs/app.module.ts:55]: This property isn't necessary.

File: test/benchmarks/mdc/tabs/fake-tab-data.ts [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+// tslint:disable:max-line-length
+
+export const threeTabs = '<mat-tab label="Tab #1"> Content #1 </mat-tab><mat-tab label="Tab #2"> Content #2 </mat-tab><mat-tab label="Tab #3"> Content #3 </mat-tab>';
```
Comment (crisbeto) [inline test/benchmarks/mdc/tabs/fake-tab-data.ts:11]: Couldn't we use a template string and format it correctly?

File: test/benchmarks/mdc/tabs/BUILD.bazel [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+        "@npm//@types/jasmine",
+    ],
+    ng_deps = [
+        "@npm//@angular/core",
+        "@npm//@angular/platform-browser",
+        "//src/material-experimental/mdc-tabs",
+    ],
+    ng_srcs = [
+        ":app.module.ts",
+        ":fake-tab-data.ts",
```
Comment (devversion) [inline test/benchmarks/mdc/tabs/BUILD.bazel:21]: Just a FYI: The usual convention is that colon is only used to indicate a Bazel target. Plain input files are without the colon.

**Review by devversion — APPROVED** [review]

**Review by crisbeto — APPROVED** [review]
LGTM

**Pull request merged**